runtime/configs/config.go

```diff
@@ -36,7 +36,10 @@ type Config struct {
 
```

Comment:
Lots of config changes here towards the end of the review. It's hard for me to evaluate if there are any issues (configs are always tricky), so I'm going to assume the ordering of things is correct and tested.

Reply:
Thank you, great point! I reverted the changes, and added an `INCOMPLETE(#858)` item for default configuration instead.